products/grafana-loki.md

```diff
@@ -22,6 +22,12 @@ identifiers:
 
 # eol(x) = releaseDate(x+1)
 releases:
+-   releaseCycle: "3.5"
+    releaseDate: 2025-04-17
+    eol: false
+    latest: "3.5.0"
+    latestReleaseDate: 2025-04-17
+
 -   releaseCycle: "3.4"
     releaseDate: 2025-02-12
     eol: false
```

Comment:
As per text underneath, only latest minor is supported, so 3.4 should be EoL now?

```suggestion
    eol: 2025-04-17
```